tests/index.html

```diff
@@ -21,18 +21,6 @@
 
     {{content-for 'body'}}
     {{content-for 'test-body'}}
-
-    <script>
-      // PhantomJS doesn't support bind until 2.x
-      // todo: once upgraded to 2.x, this should be removed.
-      Function.prototype.bind = Function.prototype.bind || function (thisp) {
-        var fn = this;
-        return function () {
-          return fn.apply(thisp, arguments);
-        };
-      };
-    </script>
-
```

Comment:
@rynam0 unfortunately I had to roll this back due to lack of support in travis-ci :(
https://github.com/travis-ci/travis-ci/issues/3225